frontend/src/wordInteractions.js

```diff
@@ -84,21 +87,34 @@ function loadEntryForm () {
 function loadTimer () {
   const gameTimerDiv = document.getElementById('game-timer-div');
   gameTimerDiv.classList.remove('hidden');
-}
-
-function runGameTimer () {
-  incrementTimerOnScreen();
-  chooseRandomWord();
+  TIMER_ID = setInterval(incrementTimerOnScreen, 1000);
 }
 
 function incrementTimerOnScreen () {
   const seconds = document.getElementById('seconds');
-  let time = parseInt(seconds.textContent, 10);
-  time += 1;
-  if (time < 10) {
-    seconds.textContent = '0' + time;
-  } else if (time < 60) {
-    seconds.textContent = time;
+  const minutes = document.getElementById('minutes');
+  const hours = document.getElementById('hours');
+
+  const startTime = new Date(game.run.created_at);
+  const currentTime = new Date();
+
+  const secs = Math.round((currentTime - startTime) / 1000) % 60;
+  secs < 10 ? seconds.textContent = 0 + secs : seconds.textContent = secs;
+  const mins = Math.round((currentTime - startTime) / 60000) % 60;
+  mins < 10 ? minutes.textContent = 0 + mins : minutes.textContent = mins;
+  const hrs = Math.round((currentTime - startTime) / 360000) % 24;
+  hrs < 10 ? hours.textContent = 0 + hrs : hours.textContent = hrs;
```

Comment:
I would probably pull this logic out into it's own function. Smart way to handle the leading 0's on the timer though :)

Reply:
good point, will do